designer/package.json

```diff
@@ -5,12 +5,14 @@
   "main": "index.js",
   "scripts": {
     "lint": "standard",
+    "fix-lint": "standard --fix",
     "build": "rollup -c",
     "watch": "rollup -c -w",
     "dev": "rollup -c -w & node index.js",
     "start": "node index.js",
     "symlink-env": "./bin/symlink-config",
-    "test": "lab -T node_modules/@xgovformbuilder/lab-babel test/.setup.js test/*.test.js  -v -r console -o stdout -r html -o unit-test.html -I version -l"
+    "test-lab": "lab -T node_modules/@xgovformbuilder/lab-babel test/.setup.js test/*.test.js  -v -r console -o stdout -r html -o unit-test.html -I version -l",
+    "test": "npm run lint && npm run test-lab && npm audit --audit-level=high"
```

Comment:
we're using yarn in CI, so it's best for us to use `yarn lint && yarn test-lab` etc. we can also run audit at install time for CI, `yarn install --audit`, so you don't have to run it when you're testing locally

Reply:
ok, cool. Fixed